sql/src/test/java/io/crate/analyze/SelectStatementAnalyzerTest.java

```diff
@@ -858,14 +871,17 @@ public void testExcept() {
         analyze("select * from users except select * from users_multi_pk");
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testArrayCompareInvalidArray() throws Exception {
+        expectedException.expect(IllegalArgumentException.class);
+        expectedException.expectMessage("invalid array expression: 'name'");
         analyze("select * from users where 'George' = ANY (name)");
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test // TODO: remove this artificial limitation in general
```

Comment:
I moved it there cause the test method could be totally removed if the limitation is removed.